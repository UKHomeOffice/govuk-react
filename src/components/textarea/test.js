import React from "react";
import ReactDOM from "react-dom";
import { Textarea } from "./";

describe("Textarea", () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Textarea>Example</Textarea>, div);
  });
});
