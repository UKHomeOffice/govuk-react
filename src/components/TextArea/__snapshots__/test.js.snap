// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Textarea matches snapshot: enzyme.mount 1`] = `
<TextArea
  hint={null}
  input={Object {}}
  meta={Object {}}
>
  <glamorous(label)>
    <label
      className="css-1dyhx6n"
    >
      <glamorous(span)>
        <span
          className="css-1m1nyko"
        >
          example
        </span>
      </glamorous(span)>
      <glamorous(textarea)
        rows="5"
        type="text"
      >
        <textarea
          className="css-1qor5e0"
          rows="5"
        />
      </glamorous(textarea)>
    </label>
  </glamorous(label)>
</TextArea>
`;

exports[`Textarea matches snapshot: enzyme.render 1`] = `
<label
  class="css-1dyhx6n"
>
  <span
    class="css-1m1nyko"
  >
    example
  </span>
  <textarea
    class="css-1qor5e0"
    rows="5"
  />
</label>
`;

exports[`Textarea matches snapshot: enzyme.shallow 1`] = `
<glamorous(label)>
  <glamorous(span)>
    example
  </glamorous(span)>
  <glamorous(textarea)
    rows="5"
    type="text"
  />
</glamorous(label)>
`;
